### PR TITLE
Fix followup visibility in applications

### PR DESCRIPTION
**Pull request by starvinraven**

### CODE REVIEW

**Comment by elahti** [comment]
It was still possible to get an invalid followup question to be visible in application handling and in Excel export. Steps to reproduce:

1. Create a application with a dropdown. One option with a followup question (eg. an input field), one option without one.
2. Open the application in the applicant side.
3. Select the dropdown option *with* the followup question. Type in something as a value to the followup question.
4. Select the dropdown option *without* the followup question.
5. Submit the application.
6. Check the application in the application handling. It has the followup question. It also appears in the Excel export.

File: src/cljs/ataru/hakija/application_handlers.cljs [hunk 1]
```diff
@@ -79,13 +79,7 @@
 (reg-event-fx
   :application/submit
   (fn [{:keys [db]} _]
-    (send-application db :submit)
```
Comment (auramo) [inline src/cljs/ataru/hakija/application_handlers.cljs]: So all of this was removed, does it still work? Did we only remove the submitting-status?

**Comment by auramo** [comment]
Couldn't create followup questions

missa-lisakysymys

Steps to reproduce: create new form, add dropdown -> no followup-controls visible.

File: src/cljs/ataru/hakija/application_handlers.cljs (hunk 1 above)
Comment (starvinraven) [inline src/cljs/ataru/hakija/application_handlers.cljs]: `send-application` has no side effects, so it's just dead code here (also there's no HTTP method named `submit`)... I removed the extracted form of the returned map and replaced it with the send-application (with `post` method) which seems to be functionally identical.

**Comment by starvinraven** [comment]
Works On My Machine™. Try clean + rebuild?

**Pull request merged**